rootfs/usr/share/www/static/scripts.js

```diff
@@ -112,8 +112,44 @@ function scheduleFetchLogs() {
   scheduleTimeout = setTimeout(fetchLogs, 5000);
 }
 
+function fetchNetworkInfo() {
+  fetch("/supervisor/network/info").then(function (res) {
+    if (!res.ok)
+      return;
```

Comment:
We should schedule a new fetch here I guess?

Reply:
It's done unconditionally now at the end of the function.